### ISSUE TITLE
[MSFT-199] All pages: Inconsistent landmarks across pages.

### ISSUE THREAD

**Issue by veshu** [importtest/test#46]
**Steps to Reproduce**
1. Go to the sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure that the screen reader is switched on. 3. Press the letter 'r' a couple of times to navigate to the various landmarks on the page or press 'ctrl + insert + r' to view the list of landmarks on the page. 4. Note that there is no 'banner' landmark for the header, 'main landmark' for the main content of the page, and 'contentinfo landmark' for the footer of the page. 5. Also note that this issue can be found on the other pages too. The landmarks are inconsistent.

**Expected Result**
The content on the page should be placed inside the relevant landmark roles for screen reader users to skip to the content in the respective landmark he/she intends to read. The banner landmark should contain the header of the page. The main landmark should be wrapped around the main content of the page. The contentinfo should be used to contain the footer of the page. Navigation, search etc can be used additionally.

**Actual Result**
There are no banner, main and contentinfo landmarks on the page.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.1 Bypass Blocks
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2034
